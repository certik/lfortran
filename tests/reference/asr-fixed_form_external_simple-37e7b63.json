{
    "basename": "asr-fixed_form_external_simple-37e7b63",
    "cmd": "lfortran --indent --fixed-form --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/fixed_form_external_simple.f",
    "infile_hash": "5fb4d1f0c148a2dcac928b0d1bde13987cd8b687962a9f98b2459ac9",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-fixed_form_external_simple-37e7b63.stderr",
    "stderr_hash": "2d66308a75e86656c0a3dcdae28e0d908e1aaaceb70d91a918f3d084",
    "returncode": 2
}