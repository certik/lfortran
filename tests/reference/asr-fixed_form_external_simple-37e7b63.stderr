semantic error: Function 'inner_scope' not found or not implemented yet (if it is intrinsic)
  --> tests/fixed_form_external_simple.f:11:11
   |
11 |           call inner_scope(b, c) ! this should fail in compilation, not earlier          
   |           ^^^^^^^^^^^^^^^^^^^^^^ 
